lnbits/core/migrations.py

```diff
@@ -692,3 +692,14 @@ async def m030_add_user_api_tokens_column(db: Connection):
         ALTER TABLE accounts ADD COLUMN access_control_list TEXT
         """
     )
+
+
+async def m031_add_color_and_icon_to_wallets(db: Connection):
+    """
+    Adds icon and color columns to wallets.
+    """
+    try:
+        await db.execute("ALTER TABLE wallets ADD COLUMN icon TEXT DEFAULT 'flash_on'")
```

Comment:
- `wallets` table should have an `extra` column where we put these additional fields (and future ones)
- see `User` class:
```py
class User(BaseModel):
    id: str
    created_at: datetime
    ...
    extra: UserExtra = UserExtra()
```

Reply:
Why, are they not just two different approaches with similar weight pros/cons? As by same logic something currency would go there as well?